react/reactHelp.md

```diff
@@ -0,0 +1,25 @@
+## How to run react prototype
+
+#### Create branch before PR(Pull Request)
+1. $ `npm install`
+2. $ `npm start`
+3. open web-browser with url `http://localhost:3000/` (can be changed)
+4. test: press arrow button (up/down/left/right)
+5. comments
+    * The bird is moving according to key input, but it's not stopping for now.
+    * The image of bird was taken from Blockly respository for test purpose.
+    * Note that you might need to clean up `node_modules` folder under `react`
+
+#### Next goals
+* refactoring structure
+* Fix bugs
+* Integrate react with Blockly
```

Comment:
check out the `react-blockly` package. I merged #17 which installs the package so if you rebase this branch you may find that a useful place to start.